src/app/admin/_components/Organization/SelectTherapies.js

```diff
@@ -0,0 +1,18 @@
+import { SelectArrayInput, required } from 'react-admin';
+import { THERAPIES } from '@admin/_lib/consts';
+import PropTypes from 'prop-types';
+
+export function SelectTherapies({ isActive }) {
+  return (
+    <SelectArrayInput
+      label="Типи"
+      source="therapies"
+      validate={isActive && required()}
+      choices={THERAPIES.map(therapy => ({ id: therapy, name: therapy }))}
```

Comment:
Therapies should also be taken from database

Reply:
ok, I'll change it all to fetching from db